scripts/init_venv.py

```diff
@@ -134,20 +132,37 @@ def link_pyqt():
         link_name = os.path.join(venv_path, fn)
         if not os.path.exists(source):
             raise FileNotFoundError(source)
+        if os.path.exists(link_name):
+            os.unlink(link_name)
         print('{} -> {}'.format(source, link_name))
         os.symlink(source, link_name)
 
 
 def create_venv():
-    """Create a new virtualenv."""
-    utils.print_title("Creating virtualenv")
-    if os.name == 'nt':
-        sys_site = ['--system-site-packages']
-    else:
-        sys_site = []
-    subprocess.check_call(['virtualenv'] + sys_site +
-                          ['-p', sys.executable, g_path])
+    """Create a new venv."""
+    utils.print_title("Creating venv")
+
+    system_site_packages = True
+
+    clear = False
+    if g_args.clear or g_args.force:
+        clear = True
+
+    upgrade = False
+    if g_args.upgrade:
+        upgrade = True
 
+    symlinks = False if os.name == 'nt' else True
+        
+    builder = venv.EnvBuilder(system_site_packages=system_site_packages,
+                              clear=clear, upgrade=upgrade,
+                              symlinks=symlinks, with_pip=True)
+
+    context = builder.ensure_directories(g_path)
+    builder.create_configuration(context)
+    builder.setup_python(context)
+    builder.setup_scripts(context)
```

Comment:
On Fri, 09 Jan 2015 10:03:37 -0800
Florian Bruhin <notifications@github.com> wrote:

> >  
> > +    symlinks = False if os.name == 'nt' else True
> > +        
> > +    builder = venv.EnvBuilder(system_site_packages=system_site_packages,
> > +                              clear=clear, upgrade=upgrade,
> > +                              symlinks=symlinks, with_pip=True)
> > +
> > +    context = builder.ensure_directories(g_path)
> > +    builder.create_configuration(context)
> > +    builder.setup_python(context)
> > +    builder.setup_scripts(context)
> 
> I think you can simply call `builder.create(g_path)` after creating the `EnvBuilder` instead of doing this.

Oh, re-reading the docs I see that as well. Will
change accordingly.